runtime/src/main/jni/MetadataNode.cpp

```diff
@@ -807,6 +814,7 @@ void MetadataNode::SetInstanceMetadata(Isolate* isolate, Local<Object> object, M
 }
 
 void MetadataNode::ExtendedClassConstructorCallback(const v8::FunctionCallbackInfo<v8::Value>& info) {
+    TNSPERF();
```

Comment:
TNSPERF uses `__func__` for name and is nothing but a shorthand for `tns::instrumentation::Frame frame(__func__);`